src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+        User userExpectedFirst = storageDao.add(USER_WITH_AGE_NULL);
+        User userExpectedSecond = storageDao.add(USER_WITH_LOGIN_NULL);
+        User userExpectedThird = storageDao.add(USER_WITH_PASSWORD_NULL);
+
+        User actualFirstResultNull = storageDao.get(USER_WITH_AGE_NULL.getLogin());
+        User actualSecondResultNull = storageDao.get(USER_WITH_LOGIN_NULL.getLogin());
+        User actualThirdResultNull = storageDao.get(USER_WITH_PASSWORD_NULL.getLogin());
+
+        assertNotEquals(userExpectedFirst, actualFirstResultNull,
+                "Test failed! Age cannot be null");
+        assertNotEquals(userExpectedSecond, actualSecondResultNull,
+                "Test failed! Login cannot be null");
+        assertNotEquals(userExpectedThird, actualThirdResultNull,
+                "Test failed! Password cannot be null");
+    }
+
+    @Test
+    public void testAddUserWithAgeLessThanMinAge_NotOk() {
+        User userExpectedFirst = storageDao.add(USER_WITH_AGE_LESS_THAN_MIN_AGE);
+        User userExpectedSecond = storageDao.add(USER_WITH_NEGATIVE_AGE);
+
+        User actualFirstResultNull = storageDao.get(USER_WITH_AGE_LESS_THAN_MIN_AGE.getLogin());
+        User actualSecondResultNull = storageDao.get(USER_WITH_NEGATIVE_AGE.getLogin());
+
+        assertNotEquals(userExpectedFirst, actualFirstResultNull,
+                "Test failed! Age cannot be less than 18");
+        assertNotEquals(userExpectedSecond, actualSecondResultNull,
+                "Test failed! Age cannot be less than 18");
+    }
+
+    @Test
+    public void testAddUserWithAgeMoreThanMinAge_Ok() {
+        storageDao.add(THIRD_USER);
+        User expected = THIRD_USER;
+        User actual = storageDao.get(THIRD_USER.getLogin());
+        assertEquals(expected, actual, "Test failed! Age cannot be less than 18");
+    }
+
+    @Test
+    public void testAddUserWithPasswordLessThanSix_NotOk() {
+
+        User userExpectedFirst = storageDao.add(
+                new User("Volodymyr", "", 25));
+        User userExpectedSecond = storageDao.add(
+                new User("Volodymyr", "abc", 25));
+        User userExpectedThird = storageDao.add(
+                new User("Volodymyr", "abcdf", 25));
+
+        User userActualFirst = storageDao.get(userExpectedFirst.getLogin());
+        User userActualSecond = storageDao.get(userExpectedSecond.getLogin());
+        User userActualThird = storageDao.get(userExpectedThird.getLogin());
+
+        assertNotEquals(userExpectedFirst, userActualFirst,
+                "Test failed! Password length cannot be less than 6");
+        assertNotEquals(userExpectedSecond, userActualSecond,
+                "Test failed! Password length cannot be less than 6");
+        assertNotEquals(userExpectedThird, userActualThird,
+                "Test failed! Password length cannot be less than 6");
+
+    }
+
+    @Test
+    public void testAddUserWithPasswordMoreThanSix_Ok() {
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/49767049/8bb6dbe4-8b89-4992-94c5-f3b3756de755)
rename according to convention